src/main/scala/uk/gov/nationalarchives/dp/client/Client.scala

```diff
@@ -0,0 +1,166 @@
+package uk.gov.nationalarchives.dp.client
+
+import cats.MonadError
+import cats.effect.Sync
+import cats.implicits._
+import com.github.benmanes.caffeine.cache.Caffeine
+import sttp.capabilities.Streams
+import sttp.client3._
+import sttp.client3.upicklejson._
+import uk.gov.nationalarchives.dp.client.Client.{AuthDetails, BitStreamInfo}
+import upickle.default._
+
+import java.time.format.DateTimeFormatter
+import java.time.ZonedDateTime
+import java.util.UUID
+import scala.xml.{Elem, XML}
+import scalacache._
+import scalacache.caffeine._
+import scalacache.memoization._
+
+import scala.concurrent.duration._
+
+trait Client[F[_], S] {
+  def metadataForEntity(entity: Entity, auth: AuthDetails): F[Seq[Elem]]
+
+  def getBitstreamInfo(contentId: UUID, authDetails: AuthDetails): F[Seq[BitStreamInfo]]
```

Comment:
It's called `contentId` here and `contentRef`, elsewhere

Reply:
Thanks, I've changed it.